subo/release/version.go

```diff
@@ -2,13 +2,13 @@ package release
 
 // SuboDotVersion represents the dot version for subo
 // it is also the image tag used for builders
-var SuboDotVersion = "0.2.0"
+var SuboDotVersion = "0.2.1"
 
 // FFIVersion is the FFI version used by this version of subo
 var FFIVersion = "0.13.0"
 
 // AtmoVersion is the default version of Atmo that will be used for new projects
-var AtmoVersion = "0.4.0"
+var AtmoVersion = "0.4.1"
```

Comment:
This should be 0.4.2

Reply:
Not sure why the diff is showing 0.4.0 as the old version, maybe merge main in? It's `0.4.2` on main.